### PR TITLE
Regenerated IoT Sdk for routing 

### PR DESCRIPTION
**Pull request by anusapan**
## Description
Regenerating IoT SDK based off of a new swagger spec.
Updated swagger [link](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/iothub/resource-manager/Microsoft.Devices/stable/2018-04-01/iothub.json)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/IotHub/Management.IotHub/Generated/IotHubResourceOperations.cs [hunk 1]
```diff
@@ -2532,23 +2531,23 @@ internal IotHubResourceOperations(IotHubClient client)
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<IotHubNameAvailabilityInfo>> CheckNameAvailabilityWithHttpMessagesAsync(OperationInputs operationInputs, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<IPage<EndpointHealthData>>> GetEndpointHealthWithHttpMessagesAsync(string resourceGroupName, string iotHubName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```
Comment (dsgouda) [inline src/SDKs/IotHub/Management.IotHub/Generated/IotHubResourceOperations.cs]: This looks like a breaking change. Please update the package version to `3.0.0`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please create a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code.

Please run `msbuild build.proj /t:build /p:Scope=SDKs\IotHub` and commit changes to the `.props`

File: src/SDKs/IotHub/Management.IotHub/Generated/IotHubResourceOperations.cs (hunk 1 above)
Comment (anusapan) [reply src/SDKs/IotHub/Management.IotHub/Generated/IotHubResourceOperations.cs]: Done

**Comment by dsgouda** [comment]
@anusapan Please merge your commits into a single commit and update the PR

**Comment by anusapan** [comment]
@dsgouda merge all commits and updated the PR

**Comment by dsgouda** [comment]
@anusapan will open a new PR with single commit

**Comment by anusapan** [comment]
Created new PR(https://github.com/Azure/azure-sdk-for-net/pull/4433) with single commit

**Pull request closed without merging**